config/archaeology/migrate_plus.migration.hs_events_importer.yml

```diff
@@ -185,14 +185,24 @@ process:
     -
       plugin: default_value
       default_value: null
+  speaker_person:
+    plugin: entity_lookup
+    source: speaker
+    value_key: title
+    bundle_key: type
+    bundle: hs_person
+    entity_type: node
+    ignore_case: true
   field_hs_event_speaker:
     -
       source: speaker
-      plugin: entity_generate_no_lookup
+      plugin: hs_entity_generate
```

Comment:
Why the override? Can you leave a code comment here?

Reply:
I think a decent comment explanation is in the plugin class. https://github.com/SU-HSDO/suhumsci/pull/72/files#diff-acab96968cfb59d1284c3f126cc3ad23R23